### PR TITLE
Add `poetry update` to restart script

### PR DESCRIPTION
**Pull request by Lunarmagpie**
Dependencies should be updated when the bot restarts since there isn't a lockfile in the repo

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 while true
 do
     git pull
+    "${1:-python}" -m poetry update
     "${1:-python}" -m poetry install
```
Comment (CircuitSacul) [inline run.sh:7]: I don't think we need install, since there's update.

Comment (MithicSpirit) [reply run.sh:7]: We should probably leave it in unless someone tests it and confirms one way or the other.

**Comment by Lunarmagpie** [comment]
I dont think `poetry update` installs the mcodingbot package, but idk if that matters.

**Review by MithicSpirit — COMMENTED** [review]
Should we update before or after installing?

**Comment by Lunarmagpie** [comment]
Before, otherwise `install` would install outdated dependencies.

**Comment by MithicSpirit** [comment]
I am still of the opinion we should keep a lockfile on some stable branch and remove it on a dev branch (which is where pull requests are made to), but whatever.

**Comment by CircuitSacul** [comment]
> I am still of the opinion we should keep a lockfile on some stable branch (which is where the bot pulls from) and remove it on a dev branch (which is where pull requests are made to), but whatever.

I'm starting to agree with you, tbh, but I think this is good for now.

**Comment by Lunarmagpie** [comment]
> I am still of the opinion we should keep a lockfile on some stable branch and remove it on a dev branch (which is where pull requests are made to), but whatever.

No reason to use poetry otherwise. We could migrate to setuptools with the current setup and it would be easier for new contributors lol.

**Review by CircuitSacul — APPROVED** [review]

**Review by MithicSpirit — APPROVED** [review]
I don't approve of removing the lockfile but whatever

**Comment by Lunarmagpie** [comment]
> I don't approve of removing the lockfile but whatever

The lockfile was gone anyway for some reason?

**Pull request merged**